### ISSUE TITLE
Azure SQL database

### ISSUE THREAD

**Issue by TajMahPaul** [chriscourses/voyager#4]
can we connect this to a SQL database hosted on Azure?

**Comment by christopher4lis** [comment]
Indeed, you should be able to connect the app to a SQL database hosted on Azure, you'll just need to make sure you deploy your app and create a .env file with credentials that link to the Azure database.
